### PR TITLE
docs: Fixed an issue where demo could not be demonstrated when started

### PR DESCRIPTION
**Pull request by fuergaosi233**
## Checklist

- [x] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [x] Tests for the changes have been added
- [x] CI has been passed. (GitHub actions all turns green)
- [ ] CLA has been signed

## Description

> please describe the changes that you are making, with the related issue number.
When I use the demo in the documentation, it does not pass the Typescript test properly.
image
Change Demo Script to fix it.

### CODE REVIEW

**Comment by huan** [comment]
Could you please explain what you have fixed in detail, and why do we need to do that?

**Comment by hcfw007** [comment]
Please see the reason for this in https://github.com/wechaty/wechaty/issues/2384

**Comment by huan** [comment]
Please make sure the following requirements have been met for your PR before we can continue reviewing it:

1. You have signed the CLA by clicking the `[CLA | not signed yet]` button from the comments.
2. The PR has passed all the CI tests and turns _green_. (you can execute all tests locally by running `npm test`)

**Comment by fuergaosi233** [comment]
> Could you please explain what you have fixed in detail, and why do we need to do that?

I'm sorry I missed this message, the problem is the same as this issue. The Bot on return type does not execute properly with the Demo, so I modified the Demo to make sure the Bot can be started.

File: README.md
Comment (huan) [inline README.md:101]: Could you please put all lines in the `scan` listener function in one line?

After that, I think this PR is good to go as a workaround for now.

**Review by huan — CHANGES_REQUESTED** [review]

File: README.md
Comment (huan) [inline README.md:101]: They are still not in one line.

We need to put all code to one line just as same as before, thanks.

File: README.md
Comment (huan) [inline README.md:98]: rename `wechatyBot` to `wechaty` to make the variable shorter and clearer.

File: README.md
Comment (huan) [inline README.md:96]: Please remove the trailing `;` because it's not the style of the Wechaty community.

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for the update.

Please fix the `;` then this PR will be ready to be merged.

File: README.md
Comment (fuergaosi233) [reply README.md]: Fixed

File: README.md
Comment (huan) [inline README.md:96]: Please use `'` instead of `"` so that your modification still follows the Wechaty coding style.

**Review by huan — CHANGES_REQUESTED** [review]

File: README.md
Comment (fuergaosi233) [reply README.md]: Fixed

File: README.md
Comment (huan) [inline README.md:99]: Remove the blank line

File: README.md
Comment (huan) [inline README.md:101]: Please keep the spaces so that the `=>` can be aligned across lines like the original style.

File: README.md
Comment (huan) [inline README.md:103]: Please remove all the trailing `;` so that it will not break the Wechaty community style.

**Review by huan — CHANGES_REQUESTED** [review]

**Comment by huan** [comment]
ping @fuergaosi233 

This PR will be ready to merge after we have fixed the above minor issues.

File: README.md
Comment (fuergaosi233) [reply README.md:103]: OK

File: README.md
Comment (huan) [inline README.md:106]: I think the above line should be removed because you have added the below line?

File: README.md
Comment (huan) [inline README.md:101]: Remove the unnecessary `{}` to make the code clear

**Review by huan — CHANGES_REQUESTED** [review]
There are two minor issues that need to be fixed before this PR can be merged.

Thanks!

File: README.md
Comment (fuergaosi233) [reply README.md]: fixed

File: README.md
Comment (fuergaosi233) [reply README.md]: I re-copied the original and modified it, I forgot to remove this line, it has been corrected.

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thank you very much for your contribution!

## You are welcome to join [Wechaty Contributor Program](https://wechaty.js.org/docs/contributor-program/)

### 1. Join Wechaty Organization

> [](https://github.com/wechaty)You've invited Holegots to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message. (See also: [wechaty/PMC#16](https://github.com/wechaty/PMC/issues/16))

### 2. Update Your Wechaty Contributor Profile

1. Please open [Contributor Hall of Fame](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0) and add yourself to the end of the list, so that other contributors will know you better!
1. Please add yourself to our Website [Contributors](https://wechaty.js.org/contributors/) by [creating a PR](https://github.com/wechaty/wechaty.js.org/tree/master/jekyll/_contributors) and refer to this PR link as well.

### 3. Join The Contributor Only WeChat Room

We also have a WeChat room for contributors only which can discuss Wechaty at a deeper level, you are welcome to join and if you are interested.

Please add @lijiarui wechat: _ruirui_0914_ and send her this pr link. She will invite you into Wechaty Contributor Room

Cheers!

**Pull request merged**
